.github/workflows/perf-testing-setup.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test_to_run: [ SyncHttpTriggerHelloWorld, SyncHttpTriggerWithSyncRequests, AsyncHttpTriggerWithAsyncRequest, SyncHttpTriggerCPUIntensive, SyncPutBlobAsBytesReturnHttpResponse, SyncGetBlobAsBytesReturnHttpResponse ]
+        test_to_run: [ SyncHelloWorld, SyncWithSyncRequests, AsyncWithAsyncRequest, SyncCPUIntensive, SyncPutBlobAsBytesReturnHttpResponse, SyncGetBlobAsBytesReturnHttpResponse ]
```

Comment:
nit: can we add these strings in some static file and check it in? and access the values here